pkg/handler/v2/routes_v2.go

```diff
@@ -321,6 +322,19 @@ func (r Routing) RegisterV2(mux *mux.Router, metrics common.ServerMetrics) {
 		Path("/projects/{project_id}/clusters/{cluster_id}/addons/{addon_id}").
 		Handler(r.deleteAddon())
 
+	// Defines a set of HTTP endpoints for managing alertmanager
+	mux.Methods(http.MethodGet).
+		Path("/projects/{project_id}/clusters/{cluster_id}/alertmanager/config").
+		Handler(r.getAlertmanager())
+
+	mux.Methods(http.MethodPost).
```

Comment:
Shouldn't be there a patch method as well? you how do you update the alermanager configs?

Reply:
So currently the alertmanager API struct only contains a config field which is a byte slice, and the user can POST the new configuration to the endpoint to update the config. But you are right, we will need a PATCH endpoint once we have more fields in alertmanger API, is that ok for you that we add that later?